.github/workflows/pre-submit.reviewdog.yml

```diff
@@ -0,0 +1,42 @@
+name: reviewdog
+on: [pull_request]
```

Comment:
declare the permissions `permissions: read-all`. It's not really needed because on PR they are read-all by default, but good practice to remember in general

Reply:
Need to make some changes here based on what I do on https://github.com/slsa-framework/slsa-github-generator/pull/214